utils/getRemoveLiquidityExtrinsic.ts

```diff
@@ -10,7 +14,7 @@ export default function getRemoveLiquidityExtrinsic(
 	tradeAssetValue: Balance,
 	coreAssetValue: Balance,
 	slippage: number
-): SubmittableExtrinsic<"promise"> {
+): CENNZnetExtrinsic | SubmittableExtrinsic<"promise"> {
```

Comment:
should we convert all of our `SubmittableExtrinsic<"promise">` to `CENNZnetExtrinsic`? given it's an implementation of `SubmittableExtrinsic<"promise">`

Reply:
keen to do this but not sure how - if I switch the type in `signAndSendTx` i get `Property 'signAndSend' does not exist on type 'CENNZnetExtrinsic'`

Reply:
@KarishmaBothara `signAndSend` should be available in `CENNZnetExtrinsic` right?

Reply:
no.. Its available with SubmittableExtrinsics
I will add a ticket in api to return decorated SubmittableExtrinsics with signViaEthWallet covered